core/arch/arm32/include/kernel/thread.h

```diff
@@ -33,8 +33,8 @@
 #endif
 
 #define THREAD_ID_0		0
-#define THREAD_ABT_STACK	0xfffffffe
-#define THREAD_TMP_STACK	0xffffffff
+
+#define STACK_THREAD_SIZE	8192
```

Comment:
No, the only reason it was platform dependent before was that the stacks where allocated in a platform dependent way. All that is unified now.